operator/roles/kiali-deploy/templates/openshift/console-links.yaml

```
@@ -0,0 +1,30 @@
+apiVersion: console.openshift.io/v1
+kind: ConsoleLink
+metadata:
+  name: kiali-app-link
```

This should not be hardcoded as a fixed string because now this breaks multi-tenancy scenarios. In other words, suppose I have control plane A and B where kiali is installed in A and B... how can I have two ConsoleLinks pointing to kiali A and B? ConsoleLinks are cluster scoped so there can only be a single link named "kiali-app-link".

We need a way to name these ConsoleLinks specific to the kiali in question.

We can use kiali_vars.deployment.namespace as a unique identifier. So I would make this name have that inside it like this:

```suggestion
  name: "kiali-app-link-{{kiali_vars.deployment.namespace }}"
```